fizzbuzz/index.html

```diff
@@ -9,6 +9,7 @@
 <body>
     <h1>Methods 2022 - FizzBuzz</h1>
     <ul>
+            <li><a href="./COBOL/nathaniel_priestley/README.md">Nathaniel's solution</a></li>
```

Comment:
I believe you'll have to resolve some conflicts here to be able to merge
Everything works properly
+10 
![image](https://user-images.githubusercontent.com/22773192/160885452-812d9138-480b-4f25-904d-499eca403c08.png)